From cd00ca90ad0d5e24278469a5da35a672d3b12ed9 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 28 Nov 2017 17:32:21 +0800
Subject: [PATCH 2902/5242] MLK-17014-1 ARM64: dts: imx8mq: move 'dcss_disp0'
 node to dtsi file

commit  6be24b9d11fcc7f9b83008f62fccf43d1dd4d4f9 from
https://source.codeaurora.org/external/imx/linux-imx.git

The 'dcss_disp0' node definition is better to be put
into the dcss node definition in 'fsl-imx8mq.dtsi'.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    8 --------
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi    |    8 ++++++++
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 732f823..3f09af1 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -673,14 +673,6 @@
 	status = "okay";
 
 	disp-dev = "hdmi_disp";
-	dcss_disp0: port@0 {
-		reg = <0>;
-
-		dcss_disp0_hdmi: hdmi-endpoint {
-			remote-endpoint = <&hdmi_disp>;
-		};
-	};
-
 };
 
 &hdmi {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index ef9201d..36dfb74 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -553,6 +553,14 @@
 				       <400000000>,
 				       <400000000>;
 		status = "disabled";
+
+		dcss_disp0: port@0 {
+			reg = <0>;
+
+			dcss_disp0_hdmi: hdmi-endpoint {
+				remote-endpoint = <&hdmi_disp>;
+			};
+		};
 	};
 
 	hdmi: hdmi@32c00000 {
-- 
1.7.9.5

